app/server.ml

```diff
@@ -80,6 +82,20 @@ type t = {
   he : Happy_eyeballs_lwt.t ;
 }
 
+let find_queue t p =
+  match SM.find_opt p t.queues with
+  | Some q -> q
+  | None ->
+    let q = Queue.create () in
+    S.iter (fun { job ; _ } ->
+      match job with
+      | Builder.Script_job { platform  ; _ } ->
+        if String.equal platform p then Queue.add job q
+      | Builder.Orb_build_job _ -> Queue.add job q)
```

Comment:
we should check that the template for this platform is available -- otherwise there's not much use in a queue that is kept in memory, but will not be used